### ISSUE TITLE
When supplying port for external services, the creation of the service fails

### ISSUE THREAD

**Issue by georgeboot** [rancher/rio#303]
When I create an external service with a specific port, using: `rio external create onedotonedotonedotone 1.1.1.1:53`, the services does not get created in the background.

When running `rio systemlogs` I get the following error:

```
rio-controller | E0614 16:29:11.090083       1 controller.go:117] error syncing 'default/onedotzero': handler stack-external-service-object-controller: failed to create default/onedotzero /v1, Kind=Service for stack-external-service default/onedotzero: Service "onedotzero" is invalid: spec.externalName: Invalid value: "1.0.0.1:53": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), requeuing
```

**Comment by georgeboot** [comment]
Additionally, is there a way to specify protocols (UDP or TCP) and multiple ports on one service?

**Comment by StrongMonkey** [comment]
This should be fixed in v0.2.0. Please reopen the issue if the latest release doesn't work for you.
